__tests__/react.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from "react"
+import { mount } from "enzyme"
+
+import createStore from "../src/index"
+import { Provider, Connect, connect } from "../src/react/index"
+
+describe("redux-zero - react bindings", () => {
+  const listener = jest.fn()
+  let store, unsubscribe
+  beforeEach(() => {
+    store = createStore({})
+    listener.mockReset()
+    unsubscribe = store.subscribe(listener)
+  })
+
+  describe("Provider", () => {
+    it("should provide the store in the apps context", () => {
+      store.setState({ message: "hello" })
+
+      class Comp extends React.Component {
+        static contextTypes = {
+          store: () => null
+        }
+        render() {
+          return <h1>{String(!!this.context.store)}</h1>
```

Comment:
Why not simply render the content of the store?

```jsx
return <h1>{this.context.store}</h1>
```

Reply:
This could work, but it would awkwardly output something like [object Object]

Reply:
I mean without the `String(this.context.store)`.
Just `<h1>{this.context.store}</h1>`

Reply:
It's kind of a cumbersome assertion, but it has a purpose that is to check if the store is in the context.

If we were to use `return <h1>{this.context.store}</h1>` the tests would break because `h1` would have an object that is alien to vdom as children. And if we were to use `String(this.context.store)` we would get `[object Object]`.

Reply:
Ok, got it. So leave it like that :+1: